### PR TITLE
Steven barrett

### PR DESCRIPTION
**Pull request by steven-barrett**
Completed everything along with most of the stretch items

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -62,33 +62,76 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
 
 // ==== Challenge 4 ====
-// The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
+// The accounting team needs all the years from every car on the lot. Create a new array from the dealer
+// data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (i = 0; i < inventory.length; i++) {
+    carYears.push(inventory[i].car_year);    
+    console.log(carYears[i]);
```
Comment (adammathieson) [inline assignments/arrays.js:101]:
```suggestion

```

File: assignments/arrays.js [hunk 2]
```diff
@@ -62,33 +62,76 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+}
```
Comment (adammathieson) [inline assignments/arrays.js:103]:
```suggestion
    console.log(carYears);
```

File: assignments/arrays.js [hunk 3]
```diff
@@ -62,33 +62,76 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+//contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the
+// results of the array in the console.
 let BMWAndAudi = [];
-console.log();
+for (i = 0; i < inventory.length; i++) {
+    if (inventory[i] === undefined) {
+        break;
+    }
+    let carMake = inventory[i].car_make;
+    if (carMake === 'Audi' || 'BMW') {
```
Comment (adammathieson) [inline assignments/arrays.js:129]:
```suggestion
    if (carMake === 'Audi' || carMake === 'BMW') {
```

Comment (adammathieson) [reply assignments/arrays.js:129]: this would be the syntax

**Review by adammathieson — COMMENTED** [review]
Great job with the objects
Couple changes to arrays, but nice job overall
Nailed the arrow function conversation

Great start to a difficult week. Really nice work

**Comment by steven-barrett** [comment]
Thanks for the advice!

**Pull request merged**